components/ArticleItem.js

```diff
@@ -7,8 +7,10 @@ export default function ArticleItem({ article }) {
   return (
     <Link route="article" params={{ id: article.get('id') }}>
       <a className="item">
```

Comment:
The `ul` element [does not support a tag](http://w3c.github.io/html/single-page.html#the-ul-element) as its direct children. How about swapping `a` and `li` to make the structure with the spec, and alter the CSS so that it looks the same?